docpages/example_code/resolved_objects.cpp

```diff
@@ -0,0 +1,46 @@
+#include <dpp/dpp.h>
+
+int main() {
+    dpp::cluster bot("token", dpp::i_default_intents);
```

Comment:
Intents is already set to default, by default. This only needs to be `dpp::cluster bot("token")` :)